kolibri_instant_schools_plugin/assets/src/views/reset-password-page/status.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+
+  <div>
+    <h1>{{ header }}</h1>
+    <p class="explanation">
+      {{ explanation }}
+    </p>
+    <div v-if="showButton" class="buttons">
+      <k-button
+        :text="$tr('homeButton')"
+        @click="$emit('close')"
+        :primary="true"
+      />
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  import kButton from 'kolibri.coreVue.components.kButton';
+  import { ResetPasswordStates as STATES } from '../../constants';
+
+  export default {
+    name: 'resetPasswordPageStatus',
+    components: {
+      kButton,
+    },
+    props: {
+      status: {
+        type: String,
+        required: true,
+      },
+    },
+    computed: {
+      header() {
+        switch (this.status) {
+          case STATES.CHECKING_TOKEN:
+            return this.$tr('validatingToken');
+          case STATES.PASSWORD_CHANGED:
+            return this.$tr('passwordChanged');
+          case STATES.LINK_EXPIRED:
+            return this.$tr('linkExpired');
+          case STATES.OTHER_ERROR:
+            return this.$tr('serverError');
+          default:
+            return '';
+        }
+      },
+      explanation() {
+        switch (this.status) {
+          case STATES.CHECKING_TOKEN:
+            return this.$tr('confirmingToken');
+          case STATES.PASSWORD_CHANGED:
+            return this.$tr('passwordSuccessfullyChanged');
+          case STATES.LINK_EXPIRED:
+            return this.$tr('needToRequestNewToken');
+          case OTHER_ERROR:
+            return this.$tr('thereWasAProblem');
+          default:
+            return '';
+        }
+      },
+      showButton() {
+        return this.status !== STATES.CHECKING_TOKEN;
+      },
+    },
+    $trs: {
+      confirmingToken: 'Confirming the validity of our password reset token…',
```

Comment:
'the' instead of 'our'? Also, would users even know what a token is?

Reply:
Haha, another intermediate state that wasn't specced out. Maybe instead of "token", it could be "link", since there is a "link expired" page.